### DISCUSSION TITLE
Welcome to LaWGPT Discussions!

### DISCUSSION THREAD

**Discussion by pengxiao-song** [pengxiao-song/LaWGPT#64] [Announcements（公告）]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kimiest** [comment]
**It's an exsiting news. Hope this community getting better and better.**

**Comment by xiang-jpg** [comment]
我希望可以在windows机器上面运行项目，可以给一个windows的部署手册吗？

**Reply by pengxiao-song** [reply]
后续项目成熟后考虑出 windows 的部署教程，目前不知道WSL2是否可以把项目启动起来，或许可以尝试下～

**Comment by sin91** [comment]
我尝试在以下的环境中启动项目：
OS: Ubuntu 22.0.4
内存: 16Gb

在尝试启动的时候发现内存很快耗尽，启动失败：
- 启动页面截图
![startup](https://github.com/pengxiao-song/LaWGPT/assets/12936796/bf8dab56-4756-4e07-9d97-a49d134c3cc0)

- 启动之后的内存占用情况
![image](https://github.com/pengxiao-song/LaWGPT/assets/12936796/ac6928ab-a831-423f-9c45-3fb22118cdd9)

请问启动项目推荐的配置是什么

**Reply by sin91** [reply]
此问题是否是因为使用的虚拟机造成的呢

**Comment by caicaivic0322** [comment]
请问在Mac本地部署运行，需要openAI的api key吗？

**Comment by caicaivic0322** [comment]
<img width="570" alt="截屏2023-05-30 15 16 59" src="https://github.com/pengxiao-song/LaWGPT/assets/17609842/e175d338-ae0a-41c3-89f6-4654ab978d51">
下载的代码的时候，下载不下来，不知道什么原因。

**Reply by sin91** [reply]
github不稳定

可以在github上直接把项目下载下来，效果是一样的。

**Comment by ChenhaoZhu** [comment]
<img width="1303" alt="截屏2023-06-10 09 11 36" src="https://github.com/pengxiao-song/LaWGPT/assets/46872981/37505ef1-c23d-4a08-83d9-69d724acf566">
<img width="1100" alt="截屏2023-06-10 09 11 47" src="https://github.com/pengxiao-song/LaWGPT/assets/46872981/5a33b6a0-d6ad-47ad-bccf-f1536e26cc19">
您好mac启动后显示这个报错，您这里知道会是什么问题吗

**Reply by Well-xu** [reply]
我也是这个问题，请问解决了吗？

**Comment by shiqiuyong** [comment]
目前这个是不能在windows上面运行吗

**Reply by shiqiuyong** [reply]
![image](https://github.com/pengxiao-song/LaWGPT/assets/138000491/c9a091b2-9356-4179-8604-48583126a8fa)

**Reply by shiqiuyong** [reply]
我启动时，遇到的错误是这样子的，不知道啥情况

**Reply by VKCHEN-711** [reply]
解决了吗 可以运行了吗

**Comment by VKCHEN-711** [comment]
你们好呀小伙伴  有搭建成功的吗？

**Comment by chiiiooo** [comment]
bash scripts/webui.sh之后，提示We couldn't connect to 'https://huggingface.co' to load this file, couldn't find it in the cached files and it looks like minlik/chinese-alpaca-plus-7b-merged is not the path to a directory containing a file named model-00002-of-00002.safetensors，怎么在hugging face上面下下来，我远程服务器的，用huggingface上面的方法不太行
